geth/mailservice/mailservice_api_test.go

```diff
@@ -18,39 +19,45 @@ func TestRequestMessagesDefaults(t *testing.T) {
 	require.InEpsilon(t, uint32(time.Now().UTC().Unix()), r.To, 1.0)
 }
 
-func TestRequestMessagesFailures(t *testing.T) {
-	ctrl := gomock.NewController(t)
-	provider := NewMockServiceProvider(ctrl)
-	api := NewPublicAPI(provider)
+func TestRequestMessages(t *testing.T) {
+	var err error
+
 	shh := whisper.New(nil)
-	// Node is ephemeral (only in memory).
-	nodeA, nodeErr := node.New(&node.Config{NoUSB: true})
-	require.NoError(t, nodeErr)
-	require.NoError(t, nodeA.Start())
+	aNode, err := node.New(&node.Config{
+		NoUSB: true,
```

Comment:
does it matter if it is false?

Reply:
I remember that in geth 1.7.x it was throwing an exception but you didn't specify in your PR so I guess I can remove it as well.